packages/builder-tools/document-model-editor/editor.tsx

```diff
@@ -36,9 +38,9 @@ import {
 } from "./utils/helpers.js";
 
 export function DocumentModelEditor() {
+  useSetPHGlobalEditorConfig(editorConfig);
```

Comment:
Should we just name this `useSetEditorConfig`?
Might make it harder to understand how it works, but we could document the method with an explanation.